Create fixed array size for threshold data

jpsmith888 opened the pull request:
Changed loadThreshold function to use fixed array size to store snap data and thresholds. Still needs to be tested w/ boards.

jpsmith888 committed:
Create fixed array size for threshold data

Changed loadThreshold function to use fixed array size to store snap data and thresholds. Still needs to be tested w/ boards.

baileyji commented:
Try something like:
data = np.array([self.getPhaseFromSnap(i) for i in xrange(nSnap)])
thresh = -threshSig*data.std(axis=1)

jpsmith888 committed:
loadThreshold loops --> vector ops

Changes for loops in loadThreshold data collection and std calculation to vector operations per Jeb's suggestion.

jpsmith888 committed:
Update loadFreqs

Put a for loop back in to handle channel indexing.
